src/displayBackend/wayland/FrameBuffer.hpp

```diff
@@ -73,7 +73,7 @@ class WlBuffer : public DisplayItf::FrameBuffer
 		return mDisplayBuffer;
 	}
 
-	void setSurface(Surface* surface);
+	void setSurface(SurfacePtr surface);
```

Comment:
Probably better to have weak ptr here as a parameter to emphasize that it will not take ownership but only temporary using?

Reply:
Done